opm/core/wells/WellsManager_impl.hpp

```diff
@@ -199,6 +199,13 @@ void WellsManager::createWellsFromSpecs(std::vector<const Well*>& wells, size_t
                 }
             }
         }
+
+        if (wellperf_data[active_well_index].empty()) {
+            OPM_MESSAGE("****Warning: there is no perforations associated with the well "
+                        << well->name() << ", the well is ignored");
```

Comment:
We should rather use OpmLog::warning() for this. The OPM_MESSAGE macro should be deprecated I think.